client/transaction-pool/graph/src/ready.rs

```diff
@@ -659,7 +659,7 @@ mod tests {
 			bytes: 1,
 			hash: 5,
 			priority: 1,
-			valid_till: u64::max_value(),	// use the max_value() here for testing.
+			valid_till: u64::MAX,	// use the max_value() here for testing.
```

Comment:
Docs for the changes in this file weren't updated. Could just be "max value" instead of "max_value()".